From 1fe66ecac485c5ba5cd3bd96db761c741ee678c3 Mon Sep 17 00:00:00 2001
From: Simon Pena <spena@igalia.com>
Date: Thu, 17 Apr 2025 01:31:19 -0700
Subject: [PATCH] [WPE] Build fix when SYSPROF_CAPTURE is disabled
 https://bugs.webkit.org/show_bug.cgi?id=291629

Reviewed by Patrick Griffis.

SKIA_OPENTYPE_SVG was nested inside the SYSPROF_CAPTURE guard, so it was
compiled out when SYSPROF_CAPTURE was off. This addresses it.

* Source/WebKit/WebProcess/wpe/WebProcessMainWPE.cpp:

Canonical link: https://commits.webkit.org/293786@main
---
 Source/WebKit/WebProcess/wpe/WebProcessMainWPE.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Source/WebKit/WebProcess/wpe/WebProcessMainWPE.cpp b/Source/WebKit/WebProcess/wpe/WebProcessMainWPE.cpp
index a06e33fd04f8..3300f3b55e15 100644
--- a/Source/WebKit/WebProcess/wpe/WebProcessMainWPE.cpp
+++ b/Source/WebKit/WebProcess/wpe/WebProcessMainWPE.cpp
@@ -45,10 +45,11 @@
 
 #if USE(SYSPROF_CAPTURE)
 #include <wtf/SystemTracing.h>
+#endif
+
 #if USE(SKIA_OPENTYPE_SVG)
 #include <skia/modules/svg/SkSVGOpenTypeSVGDecoder.h>
 #endif
-#endif
 
 namespace WebKit {
 using namespace WebCore;
-- 
2.43.0

